spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index', type: :feature do
+  before :each do
+    @user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from US.',
+                        posts_counter: 0)
+    @post = Post.create(author_id: @user.id, title: 'Hello', text: 'This is my first post.', comments_counter: 0,
+                        likes_counter: 0)
+    visit root_path
+  end
+
+  it 'I can see the username of all other users' do
+    expect(page).to have_content('Lilly')
+  end
+
+  it 'I can see the profile picture for each user' do
+    expect(page).to have_css("img[src*='https://unsplash.com/photos/F_-0BxGuVvo']")
+  end
+
+  it 'I can see the number of posts each user has written' do
+    expect(page).to have_content 'Number of posts: 1'
+  end
```

Comment:
- [ ] @Team Great work so far, however, you still have not implemented the suggested changes,  the tests should be checking if all users' images, usernames, and numbers of posts are working, kindly consider checking the loop example to learn how to fix the test cases
![image](https://github.com/umair-rao/New-Blog-App/assets/61408860/7045ad39-6e81-47cd-8d1e-e27a6436f6cc)

Reply:
Added please review